modules/toolchains_arm_gnu/metadata.json

```diff
@@ -8,12 +8,13 @@
         }
     ],
     "repository": [
-        "github:hexdae/bazel-arm-none-eabi"
+        "github:hexdae/toolchains_arm_gnu"
```

Comment:
nit: maybe we can keep the old one so that the repository check for older versions won't fail?

Reply:
Github should keep the old link around for a while, but I was hoping we could change it here to avoid disruption going forward?

Since both links point to the same thing, will the check on older versions fail?

Reply:
The check is done in this script:
https://github.com/bazelbuild/bazel-central-registry/blob/8283b860a280f5eea7cce8a1c2c3a66d3fb5eaca/tools/bcr_validation.py#L140

which doesn't resolve the repo.

Note that this is a list, so you can have both of them here.